src/nvapi_interface.cpp

```diff
@@ -6,13 +6,46 @@
 extern "C" {
     using namespace dxvk;
 
+    static const auto disabledEnvName = "DXVK_NVAPI_DISABLE_ENTRYPOINTS";
+    static const auto disabledString = str::fromnullable(std::getenv(disabledEnvName));
+    static const auto disabled = str::split<std::set<std::string_view, str::CaseInsensitiveCompare<std::string_view>>>(disabledString, std::regex(","));
+
+    static std::once_flag logDisabledOnceFlag;
+
     static std::unordered_map<NvU32, void*> registry;
 
     __declspec(dllexport) void* __cdecl nvapi_QueryInterface(NvU32 id) {
         auto entry = registry.find(id);
         if (entry != registry.end())
             return entry->second;
 
+        std::call_once(logDisabledOnceFlag, []() {
```

Comment:
May be extract the lambda into a separate function to not distract to much from the important things in `nvapi_QueryInterface`?